lambdas/app.js

```diff
@@ -38,8 +37,23 @@ const originRequest = async (event, context) => {
     });
 };
 
-const originResponse = async (event) => {
-  const { response } = event.Records[0].cf;
+const originResponse = async (event, context) => {
+  const { request, response } = event.Records[0].cf; 
+  if (['404', '403'].includes(response.status)) {
+    const host = Utils.getHost(request);
+    const sitePath = getSitePath(request);
+    const params = getBuildQueryParams(host, sitePath, context.functionName);
+    const build = await getSiteItem(params);
+    const { Settings: { spa } } = build;
```

Comment:
does this imply that the format for specifying this in the federalist.json file is:
```
{
  ...
  spa: <anything>,
  ...
}
```
?

Reply:
@davemcorwin  - yes

Reply:
Can we make this something a little more explicit? It would be confusing if `spa: "false"` means SPA routing is ON.